frontend/packages/ceph-storage-plugin/.eslintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": ["../.eslintrc.js"],
```

Comment:
Extend on the `eslint-plugin-console`

Reply:
Fixed at https://github.com/openshift/console/pull/8131/commits/142af325d361bfaba9878faa6abcbbadee0085f5#diff-e77a554f3d40cd16fe423926aef6d9fc75d90579de71b65f621a5302794396ddR3. PTAL.